connect/runtime/src/test/java/org/apache/kafka/connect/runtime/rest/RestServerTest.java

```diff
@@ -185,6 +188,36 @@ public void testAdvertisedUri() {
         Assert.assertEquals("http://plaintext-localhost:4761/", server.advertisedUrl().toString());
     }
 
+    @Test
+    public void testValidateUriHost() {
+        Map<String, String> configMap = new HashMap<>(baseWorkerProps());
+        configMap.put(WorkerConfig.LISTENERS_CONFIG, "http://localhost:8080,https://localhost:8443");
+        DistributedConfig config = new DistributedConfig(configMap);
+
+        server = new RestServer(config);
```

Comment:
Nit: seems a little strange to have to instantiate a `RestServer` with one set of REST listeners and no advertised listeners, but then use it to validate a different set of advertised listeners. Would it make sense to change `RestServer::validateUriHost` to a `static` method to make the testing here cleaner?

Reply:
@C0urante I made it static and I Agree it made the testing clean.